src/main/web_src/src/api/index.ts

```diff
@@ -21,8 +24,15 @@ const _fetch = (url: string, method: Method, body?: object): Promise<Response> =
 			throw error
 		})
 
-const fetchJson = <T>(url: string, method: Method, body?: object): Promise<T> =>
-	_fetch(url, method, body).then((response: Response) => response.json() as Promise<T>)
+const fetchJson = <T>(url: string, config: Config, body?: object): Promise<T> =>
+	_fetch(
+		url,
+		{
+			method: config.method,
+			headers: { ...config.headers, 'Content-Type': 'application/json' }
+		},
+		body
+	).then((response: Response) => response.json() as Promise<T>)
```

Comment:
Det ser ut som get, post og put fungerer, men delete gir `Uncaught (in promise) SyntaxError: Unexpected end of JSON input`. Det fungerer i t2.

Reply:
Hmm ok skal se på det. Hvordan kan jeg gjenskape i Dolly?

Reply:
Prøv å slette en opprettet person